k8s/aat/common/divorce/div-dn.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/div/dn:prod-947a296f
+      image: hmctspublic.azurecr.io/div/dn:latest
```

Comment:
Did someone confirm this is okay? :)

Reply:
@jacmurphy Looking at the commit history, the :latest tag has never been used. Might want to verify this

Reply:
It was the solution that Dale was given by Luigi - https://hmcts-reform.slack.com/archives/C8SR5CAMU/p1616592447229600?thread_ts=1616590641.226700&cid=C8SR5CAMU

Reply:
Ok, its this the way to go for this then